### PR TITLE
Fix checks and tests for client.get_*()s

### PR DESCRIPTION
**Pull request by convoliution**
Followup to https://github.com/VertaAI/modeldb/pull/966

### CODE REVIEW

File: client/verta/tests/test_entities.py [hunk 1]
```diff
@@ -323,8 +323,9 @@ def test_get(self, client):
         assert run.id == client.get_experiment_run(run.name).id
         assert run.id == client.get_experiment_run(id=run.id).id
 
-        # clear client state to test setting parent
-        client.proj = client.expt = None
+        # test parents are restored
+        client.set_project()
```
Comment (conradoverta) [inline client/verta/tests/test_entities.py:327]: Shouldn't these be restored automatically?

Comment (convoliution) [reply client/verta/tests/test_entities.py:327]: Yes. This test is setting a new Project and Experiment, and then testing that `get_experiment_run()` restores the _old_ ones.

File: client/verta/tests/test_entities.py [hunk 2]
```diff
@@ -323,8 +323,9 @@ def test_get(self, client):
         assert run.id == client.get_experiment_run(run.name).id
         assert run.id == client.get_experiment_run(id=run.id).id
 
-        # clear client state to test setting parent
-        client.proj = client.expt = None
+        # test parents are restored
```
Comment (convoliution) [inline client/verta/tests/test_entities.py]:
```suggestion
        # test parents are restored by first setting new, unrelated ones
```

File: client/verta/tests/test_entities.py (hunk 1 above)
Comment (conradoverta) [reply client/verta/tests/test_entities.py:327]: Fair enough.

**Review by conradoverta — APPROVED** [review]

**Pull request merged**